cfme/tests/configure/test_access_control.py

```diff
@@ -690,3 +690,48 @@ def _delete_tenant_and_project():
         project.name = project.name + "edited"
     project.delete()
     tenant.delete()
+
+
+@pytest.mark.uncollectif(lambda: version.current_version() < "5.5")
+@pytest.mark.parametrize('number_of_childrens', [5])
+def test_superadmin_child_tenant_crud(request, number_of_childrens):
+    """Test CRUD operations for CFME child tenants, where several levels of tenants are created.
+
+    Prerequisities:
+        * This test is not depending on any other test and can be executed against fresh appliance.
+
+    Steps:
+        * Create 5 tenants where the next tenant is always child to the previous one
+        * Update description of tenant(N-1)_* in the tree
+        * Update name of tenant(N-1)_*
+        * Delete all created tenants in reversed order
+    """
+
+    tenant = None
+    tenant_list = []
+
+    @request.addfinalizer
+    def _delete_tenants():
+        # reversed because we need to go from the last one
+        for tenant in reversed(tenant_list):
```

Comment:
Yes, mfalesni is right, we have to delete tenants in reverse order. When there is group assigned to any child tenant (this is not case for our test, we do not create/assign groups), deletion of top tenant will fail.   
https://bugzilla.redhat.com/show_bug.cgi?id=1292157

I will try to implement check for existing groups/users assigned to tenant in the future. These would be deleted first before deleting tenant.